src/cpp/fastdds/domain/DomainParticipantImpl.hpp

```diff
@@ -559,7 +559,7 @@ class DomainParticipantImpl
     std::string get_inner_type_name(
             const fastrtps::rtps::SampleIdentity& id) const;
 
-    static void set_qos(
+    static bool set_qos(
```

Comment:
It'd be nice to have some doxygen here. Furthermore, there is a mock of this function which signature we should update

Reply:
Nice catch!